addons/binding/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/internal/PentairPacket_Intellichlor.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pentair.internal;
+
+/**
+ * Pentair Intellichlor specialation of a PentairPacket. Includes public variables for many of the reverse engineered
+ * packet content. Note, Intellichlor packet is of a different format and all helper functions in the base PentairPacket
+ * may not apply.
+ *
+ * This packet can be a 3 or 4 data byte packet.
+ *
+ * 10 02 50 00 00 62 10 03
+ * 10 02 00 01 00 00 13 10 03
+ *
+ * @author Jeff James - initial contribution
+ *
+ */
+public class PentairPacket_Intellichlor extends PentairPacket { // 29 byte packet format
```

Comment:
Naming, get rid of the underscore, please rename file and class using the refactor option in your IDE